media/js/firefox/hello/start.js

```diff
@@ -25,7 +28,32 @@
 
     // GA event for successful Hello conversations
     function trackGAConversationConnect() {
+        // get source from localStorage
+        try {
+            tourSource = localStorage.getItem('hello_ftu_tour_source');
+        } catch (ex) {
+            tourSource = 'none';
+        }
+        gaTrack(['_setCustomVar', 13, 'Hello FTU Referral', tourSource, 2]);
         gaTrack(['_trackEvent', '/hello/start interactions', 'tour', 'TourConnectConversation']);
+        try {
+            localStorage.setItem('hello_ftu_tour_source', 'none');
```

Comment:
I was thinking, maybe it's better to clean up after ourselves and remove the item from localStorage here? `localStorage.removeItem('hello_ftu_tour_source');`